spec/features/user_views_tattoo_details_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+feature "User views a specific tattoo", %(
+  As a user
+  I want to view a tattoo's details page
+  So that I could read details and reviews on that tattoo
+
+  Acceptance Criteria
+  [ ] I must visit the url of the tattoo.id
+  [ ] I must see the tattoo's title
+  [ ] I must see the tattoo's description, if one exists
+  [ ] I must see the picture of the tattoo that the url directs to
+) do
+
+  let!(:tattoo) do
+    FactoryGirl.create(:tattoo, title: 'Dagron')
+  end
+  scenario 'visitor views tattoo details' do
+    visit tattoo_path(tattoo)
+    expect(page).to have_content('Dagron')
+    expect(page).to have_content('Great')
+    expect(page).to have_css("img[src*='http://www.clipartbest.com/cliparts/4T9/xK9/4T9xK9eTE.jpeg']")
```

Comment:
This line is a little confusing as to what's happening. Instead of hard-coding the image url, could you call `.url` on the `tatoo` variable instead? I.e.

```
"img[src*=#{tattoo.url}]"
```